src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -840,6 +840,24 @@ public function findDefinition($id)
      */
     public function createService(Definition $definition, $id, $tryProxy = true)
```

Comment:
``createService`` should be marked as internal actually. It was public by necessity to support PHP 5.3, as it is called inside the closure of the proxy instantiation for lazy services